Sources/GeotabMobileSDK/Classes/DriveViewController.swift

```diff
@@ -646,6 +652,39 @@ extension DriveViewController {
     }
 }
 
+extension DriveViewController {
+    /**
+     Registers a given ScriptInjectable object with the content controller, allowing it to be executed within the web view.
+
+     The ScriptInjectable object encapsulates the source code, injection time, and message handler name for the custom script. By registering it, the custom script will be injected into the web view at the specified injection time and be able to communicate with native code using the provided message handler name.
+
+     - Parameter scriptInjectable: The script injectable object containing the properties and logic needed to define and control the custom script.
+    */
+    public func registerScriptInjectable(_ scriptInjectable: ScriptInjectable) {
+        let userScript = WKUserScript(source: scriptInjectable.source,
+                                      injectionTime: scriptInjectable.injectionTime,
+                                      forMainFrameOnly: true)
+        self.contentController.addUserScript(userScript)
+        self.contentController.add(self, name: scriptInjectable.messageHandlerName)
+        self.scriptInjectables[scriptInjectable.messageHandlerName] = scriptInjectable
+    }
+    
+    /**
+     Unregisters a given script injectable object from the content controller, removing it from execution within the web view.
+
+     If the provided script injectable object is not found in the registered scripts, the method returns early without making any changes.
+
+     - Parameter scriptInjectable: The script injectable object to be unregistered.
+    */
+    public func unregisterScriptInjectable(_ scriptInjectable: ScriptInjectable) {
+        let handlerName = scriptInjectable.messageHandlerName
+        guard scriptInjectables[handlerName] != nil else { return }
```

Comment:
I think the recommended practice for this is "let _ = " rather than "!= nil".

Reply:
I did it that way first but the reverted to "!= nil", don't really know why exactly. Found a good read also (from Swift 4.0) about this: https://albinek.com/to-if-let-or-to-if-not-nil. "guard/if let _" seems faster in all scenarios. Thanks for pointing this out, will update.